tools/windows/install-help/cal/FinalizeInstall.cpp

```diff
@@ -64,6 +64,19 @@ bool updateYamlConfig(CustomActionData &customActionData)
     return true;
 }
 
+bool writeInstallInfo(const CustomActionData &customActionData)
```

Comment:
Can we add unit tests for (1) when there was no file present and (2) when there was an install info file present (like with chocolatey)?